database/tables/inventory.sql

```diff
@@ -1,4 +1,4 @@
-DROP TABLE IF EXISTS [dbo].[Transactions]; -- Has a Foreign Key constraint on Items. 
+DROP TABLE IF EXISTS [dbo].[Transactions]; -- Has a Foreign Key constraint on Items.
 DROP TABLE IF EXISTS [dbo].[Items];
 DROP VIEW IF EXISTS InventoryWithCategory;
```

Comment:
```suggestion
DROP VIEW IF EXISTS ItemsWithCategory;
```

Reply:
There is another mention of InventoryWithCategory in cleanup.sql that needs to be renamed.